executor/aggregate_test.go

```diff
@@ -63,6 +63,7 @@ func (s *testSuite) TestAggregation(c *C) {
 	}()
 	tk := testkit.NewTestKit(c, s.store)
 	tk.MustExec("use test")
+	tk.MustExec("set sql_mode = 'STRICT_TRANS_TABLES,NO_ENGINE_SUBSTITUTION'")
```

Comment:
Why should we set sql_mode here?

Reply:
it's for make the tests below work ok. Because the default sql_mode contains "ONLY_FULL_GROUP_BY", the tests below would meet error.